module/module-registration.mjs

```diff
@@ -43,3 +49,60 @@ function registerSpellLists(manifest) {
   manifest.flags.dnd5e.spellLists.forEach(uuid => dnd5e.registry.spellLists.register(uuid));
   return "spell lists";
 }
+
+/* -------------------------------------------- */
+/*  Compendium Packs                            */
+/* -------------------------------------------- */
+
+/**
+ * Apply any changes to compendium packs during the setup hook.
+ */
+export function setupModulePacks() {
+  log("Setting Up Compendium Packs", { level: "groupCollapsed" });
+  for ( const pack of game.packs ) {
+    if ( pack.metadata.type === "Item" ) pack.applicationClass = ItemCompendium5e;
+    try {
+      const complete = setupMethods.map(m => m(pack)).filter(r => r);
+      if ( complete.length ) log(`Finished setting up ${pack.metadata.label}: ${complete.join(", ")}`);
+    } catch(err) {
+      log(`Error setting up ${pack.metadata.label}\n`, { extras: [err.message], level: "error" });
+    }
+  }
+  console.groupEnd();
```

Comment:
Feel pretty strongly about not using log groups. Too often they hide valuable errors during package development.

Reply:
Well, everything in here is synchronous system code, so there isn't really any chance of it hiding module errors.